ui/packages/consul-ui/mock-api/v1/internal/ui/oidc-auth-methods

```diff
@@ -16,8 +16,8 @@ return `
     "Name": "${name.split(' ').join('-').toLowerCase()}",
     "DisplayName": "${name}",
     "Kind": "${fake.helpers.randomize(['no-icon', 'google', 'okta', 'auth0', 'microsoft'])}",
-    "Namespace": "default",
-    "Partition": "default"
+    "Namespace": "${typeof location.search.ns !== 'undefined' ? location.search.ns : 'default'}",
+    "Partition": "${typeof location.search.partition !== 'undefined' ? location.search.partition : 'default'}"
```

Comment:
This means the mocks return auth methods for the request namespace/partition not just default all the time.